lxd/db/db.go

```diff
@@ -463,6 +466,48 @@ func TxCommit(tx *sql.Tx) error {
 	return err
 }
 
+// DqliteLatestSegment returns the latest segment ID in the global database.
+func DqliteLatestSegment() (string, error) {
```

Comment:
@MathieuBordere is there a better built-in way to get this out of dqlite?

Reply:
There currently is no API to get the latest index out of dqlite. Remark that if there are `open-xxx` files, it is possible that the latest index is not deducible from the names of the closed segment files (`0000xxxxx-0000xxxxx`). The dqlite `.reconfigure` command closes any open segment files and adds their content to a closed segment file, after that command has run, you know which node has the largest index. The command will have also created an extra index for a log entry containing the new configuration passed to the .reconfigure command.

see https://github.com/canonical/go-dqlite/blob/master/docs/restore-db.md#a3-finding-the-node-with-the-most-up-to-date-data

There's also a remark in that linked doc that I should add a command to get the latest index out of dqlite / raft.